addon/components/new-osf-navbar/template.hbs

```diff
@@ -0,0 +1,75 @@
+<div class="new-osf-navbar-styling">
+    <nav class="navbar navbar-inverse" id="navbarScope">
+        <div class="container">
+            <div class="navbar-header">
+                {{!Toggle navigation button for xs screens}}
+                <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target=".secondary-navigation" {{action 'closeSearch'}}>
+                    <span class="sr-only">Toggle navigation</span>
+                    <span class="icon-bar"></span>
```

Comment:
Actually, for this case here you make a fair comment- I may have gone through and tagged bare icons en masse. ARIA labels are the default mode for accessibility where `sr-only` is more bootstrap specific.

I'll add this link here for this specific case and let you sort out which is most appropriate for this line (answer may not be an "or"). Am curious how this turns out!
http://stackoverflow.com/questions/39223952/accessibility-sr-only-or-aria-label